### ISSUE TITLE
Use the topnav as quicklinks to the projects included in the documentation site

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-docs-ui#11]
There's going to be a lot of content on this site, so we want to make it easy for a reader to get around. One way we can do this is by using the topnav as a high-level site map as an alternative to the more detailed component explore drawer. The topnav would link to the start page for each project included in the documentation site.

The top-level may look something like this:

```
AsciiDoc   |    Plugins   |   Extensions
```

These categories would expand as follows:

* AsciiDoc
  * Language
  * Processors
     * Asciidoctor
     * AsciidoctorJ
     * Asciidoctor.js
* Plugins
   * Build Tools
      * Asciidoctor Maven Tools
      * Asciidoctor Gradle plugin
    * Browsers
         * Asciidoctor.js Live Preview
    * IDEs
         * AsciiDoc IntelliJ Plugin
* Extensions
   * Converters
      * Asciidoctor PDF
      * Asciidoctor EPUB3
      * Asciidoctor reveal.js
   * Syntax
      * Asciidoctor Diagram
      * Asciidoctor Chart

We could split Converters and Extensions.

```
AsciiDoc   |    Plugins   |   Converters   |   Extensions
```

Alternately, we could split out the Processors from underneath AsciiDoc.

```
AsciiDoc   |    Processors   |   Plugins   |   Extensions
```

**Comment by abelsromero** [comment]
How far are we going to go? There are other tools not listed like Gulp.js, Ant, asciidoctorj-*wrappers* no name a few and I don't think there's room for all in a single drop down menu.

**Comment by mojavelinux** [comment]
The goal is to link to docs in this site (as a navigation tool). We'll have
a separate page that lists all the projects in the community.

**Comment by Mogztter** [comment]
I think AsciiDoc should have its own entry, maybe with a different color. I also think that "Converters" should be a top-level entry. Or at least, not below "Extensions"

```
AsciiDoc   |   Processors   |   Plugins   |   Converters
```

* AsciiDoc
* Processors
  * Asciidoctor
  * AsciidoctorJ
  * Asciidoctor.js
* Plugins
  * Build Tools
    * Asciidoctor Maven Tools
    * Asciidoctor Gradle plugin
  * Browsers
    * Asciidoctor.js Live Preview
  * IDEs
    * AsciiDoc IntelliJ Plugin
* Converters
  * Asciidoctor PDF
  * Asciidoctor EPUB3
  * Asciidoctor reveal.js

I'm a bit torned about "Extensions"... I think it's worth noting that the difference between plugins and extensions might be unclear. For instance, the Chrome Web Store is using the term "Extensions". Some users are also using indistinctly "plugins" or "extensions".

In other words, I don't think we should have "Plugins" and "Extensions" at the same level, it will be too confusing.

Maybe we should talk about "Processor Extensions" or "Syntax Extensions"?
Does it make sense to add "Syntax Extensions" in "Processors"?

* Processors
  * Asciidoctor
  * AsciidoctorJ
  * Asciidoctor.js
  * Syntax Extensions
     * Asciidoctor Diagram
     * Asciidoctor Chart

**Comment by Mogztter** [comment]
Somehow related, we could use something similar to what Neo4j is doing to introduce the ecosystem:

![ecosystem](https://user-images.githubusercontent.com/333276/100769548-7f924680-33fc-11eb-9e05-c6b436d0bad0.png)

https://neo4j.com/developer/get-started/

**Comment by mojavelinux** [comment]
> I don't think we should have "Plugins" and "Extensions" at the same level, it will be too confusing.

But to us, they are very different things. We should not limit our choices by how a term is used somewhere else. In Asciidoctor, extensions means something very specific.

Instead of "Plugins" though, we could consider using "Integrations", which would broaden the scope a bit more.

**Comment by mojavelinux** [comment]
> Maybe we should talk about "Processor Extensions" or "Syntax Extensions"?

I think that would be getting too nuanced too early. But I could get behind something like "Extended Syntax" (a subtle difference, but an important one).

**Comment by Mogztter** [comment]
> Instead of "Plugins" though, we could consider using "Integrations", which would broaden the scope a bit more.

I quite like "Integrations", it removes the ambiguity about the "Plugins"/"Extensions" terminology.

> But to us, they are very different things. We should not limit our choices by how a term is used somewhere else. In Asciidoctor, extensions means something very specific.

I agree "extensions" means something very specific in Asciidoctor but in this case the word is used without any context (because it's used as a single word on a top-level menu).
Someone (me included) could expect to find the "Asciidoctor Browser Extension" in this menu.

> I think that would be getting too nuanced too early. But I could get behind something like "Extended Syntax" (a subtle difference, but an important one).

"Extended Syntax" is :ok_hand:

**Comment by mojavelinux** [comment]
I'm going to move forward with creating a PR and I can update as we discuss. That way, we can work with a preview in hand.

**Comment by mojavelinux** [comment]
PR is ready.
